jina/resources/Python.gitignore

```diff
@@ -78,7 +78,7 @@ docs/_build/
 
 # PyBuilder
 .pybuilder/
-target/
+# target/
```

Comment:
i suggest we keep the original ones

Reply:
I really think `target` directory is a general one. If we ignore this directory, it will cause some problems.